fiftyone/utils/sapiens.py

```
@@ -0,0 +1,296 @@
+
+sapiens = fou.lazy_import("sapiens", callback=_ensure_sapiens)
+_sapiens_pose_models = fou.lazy_import(
+    "sapiens.pose.models", callback=_ensure_sapiens
+)
+_sapiens_pose_datasets = fou.lazy_import(
+    "sapiens.pose.datasets", callback=_ensure_sapiens
+)
+
+
+@contextlib.contextmanager
+def _chdir(path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_chdir`

(ANN202)